scripts/scrubbing/validator.py

```diff
@@ -30,7 +30,6 @@
 
 import validator_data_LEGAL_REVIEW_REQUIRED # pull in the data we need to configure this tool
 sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', 'build', 'package'))
```

Comment:
this line can also go if we are not importing `glob_to_regex` anymore

Reply:
Oh how I didn't remove this line